### DISCUSSION TITLE
Button to Toggle Locking Canvas in Place

### DISCUSSION THREAD

**Discussion by thismarty** [flxzt/rnote#1129] [Ideas]
Until palm rejection is all osrted out, it sure would be nice to have a button on the toolbar trhat would lock/unlock the canvas position.  It's fairly common in my testing to see the canvas shift a little just before I begin a new stroke, triggered by my 'palm' resting on the screen. Thanks!

**Comment by Doublonmousse** [comment]
That is pretty similar to #1055
